SL-762 Add 20241028-void-mch-patient-program changeset for SL

lnball opened the pull request:
Added this changeset to the SL config.  

Unfortunately the changeset didn't run on my local SDK and I don't know why.  I searched in the log AND the liquibasechangelog table.

Feel free to review my code AND explain why it didn't run locally @mogoodrich

mogoodrich commented:
Code looks right?  If it didn't run, the two main things I could think of is: 1) error running it, but then you should have seen something in the log, or 2) somehow your change didn't get installed, check out ~/openmrs/[serverId]/configuration/pih/liquibase/iquibase-sierra_leone.xml and see if the right version of the file is there?

lnball commented:
@mogoodrich Thanks for your guidance.  The liquibase was NOT installed in the SDK.  Fixed that.  And then the sql had a stray semicolon so fixed that too.  I updated the id so there wasn't a checksum problem.  Merged to master.
